name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/BarclaysPDFExtractor.java

```diff
@@ -53,8 +77,8 @@ private void addCreditcardStatementTransaction()
 
                         .section("date", "note", "amount") //
                         .documentContext("currency") //
-                        .match("^[\\d]{2}\\.[\\d]{2}\\.[\\d]{4} (?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4})" //
-                                        + "(?<note>.*) +([A-Z]{2} Visa)( B)?( A)? +" //
+                        .match("^[\\d]{2}\\.[\\d]{2}\\.[\\d]{4} (?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) " //
+                                        + "(?<note>.{1,36}).+ +" //
```

Comment:
I think this regulare expression is not good.

Reply:
Note seems to have a fixed length (max 36 characters) and is followed by whitespaces. For now it was working in all cases. I will monitor it in the following months.

Reply:
Hi. I hope I understood your concerns correctly, that the combination of two '.' followed by each other allow multiple outcomes. For now it was working, but it could also happen, that the first character is catched into 'note'-subgroup and rest is lost.

I updated PR. Did it become worse or better?🤣
BR Christian